airbyte-integrations/bases/connector-acceptance-test/unit_tests/test_core.py

```diff
@@ -277,30 +278,94 @@ def test_configured_catalog_fixture(mocker, test_strictness_level, configured_ca
 
 
 @pytest.mark.parametrize(
-    "schema, record, expectation",
+    "schema, ignored_fields, expect_records_config, record, expected_records_by_stream, expectation",
     [
-        ({"type": "object"}, {"aa": 23}, does_not_raise()),
-        ({"type": "object"}, {}, does_not_raise()),
+        (
+            {"type": "object"},
+            {},
+            ExpectedRecordsConfig(path="foobar"),
+            {"aa": 23}, {}, does_not_raise()
```

Comment:
```suggestion
            {"aa": 23}, 
            {}, 
            does_not_raise()
```
Nit: just for clarity/consistency

Reply:
done